compat/main.py

```diff
@@ -122,17 +112,28 @@ class GoTest:
     path: str
     command: str
 
-    def run(self, working_dir: Path, prepare: bool, go_test_ref: Optional[str]) -> bool:
-        if go_test_ref:
-            replacement = f'github.com/onflow/cadence@{go_test_ref}'
+    def run(self, working_dir: Path, prepare: bool, cadence_version: Optional[str], flowgo_version: Optional[str]) -> bool:
+        if cadence_version:
+            cadence_replacement = f'github.com/onflow/cadence@{cadence_version}'
+        else:
+            cadence_replacement = Path.cwd().parent.absolute().resolve().as_posix()
+
+        if flowgo_version:
+            flowgo_replacement = f'github.com/onflow/flow-go@{flowgo_version}'
         else:
-            replacement = shlex.quote(str(Path.cwd().parent.absolute()))
+            # default flow-go version is 0.37.16 the one corresponding to cadence v1.0.0
+            flowgo_replacement = f'github.com/onflow/flow-go@v0.37.16'
```

Comment:
This will quickly get out of date and will have to get updated.

It might be better to just make this required. Even better, we could also automate this: By leveraging Go's package manager proxy, we can fetch the latest release: Fetching https://proxy.golang.org/github.com/onflow/flow-go/@latest we can use the `Version` field.

Reply:
Replaced this with `go get ...flow-go@latest`.

Reply:
Ah right, given we just run go get anyway, that works and is much simpler 👌